addons/test_website_modules/tests/test_controllers.py

```diff
@@ -89,10 +94,14 @@ def modify(login, name, expect_fail=False):
 
         # Portal user cannot modify page
         with mute_logger('odoo.http'):
-            json = modify('portal', 'page-portalfail.gif', True)
-        self.assertEqual('odoo.exceptions.AccessError', json['error']['data']['name'], "Expect access error")
```

Comment:
Why did this work before and not anymore? Could you add that info in the commit message? :thinking:

Reply:
Still would have been nice to check, maybe there is a larger bug, but as you wish